### PR TITLE
Replaces "specflags" with "species_traits"

### PR DESCRIPTION
**Pull request by ChangelingRain**
THESE AREN'T FLAGS THIS IS A LIST AAAAAAAAAAAAAAA*AAAAAAAAAAAAAAA**AAAAAAAAAAAAAAA***

### CODE REVIEW

File: code/__DEFINES/flags.dm [hunk 1]
```diff
@@ -49,7 +49,7 @@
 #define PASSMOB			16
 #define LETPASSTHROW	32
 
-//flags for species
+//flags for species //these aren't fucking flags you idiot
```
Comment (optimumtact) [inline code/__DEFINES/flags.dm]: not necessary and just confusing now

**Review by optimumtact — APPROVED** [review]

File: code/__DEFINES/flags.dm (hunk 1 above)
Comment (Xhuis) [inline code/__DEFINES/flags.dm]: 👌

**Comment by TechnoAlchemisto** [comment]
Why wouldn't you just replace the incorrect comment with a correct one instead of commenting the comment

**Comment by ChangelingRain** [comment]
it's still in flags.dm until I put it somewhere else that doesn't imply it's actually flags, it's not flags

**Pull request merged**